packages/remix-dev/compiler/loaders.ts

```diff
@@ -5,6 +5,7 @@ export const loaders: { [ext: string]: esbuild.Loader } = {
   ".aac": "file",
   ".avif": "file",
   ".css": "file",
+  ".csv": "file",
```

Comment:
We should have an entry in `packages/remix-dev/modules.ts` as well

Reply:
@MichaelDeBoey sorry for late reply, I'm already add `.csv` into `packages/remix-dev/modules.ts`